src/AltinnCore/Designer/ViewModels/Request/CreateReleaseRequestViewModel.cs

```diff
@@ -0,0 +1,38 @@
+using System.ComponentModel.DataAnnotations;
+using Newtonsoft.Json;
+
+namespace AltinnCore.Designer.ViewModels.Request
+{
+    /// <summary>
+    /// Viewmodel for creating a release
+    /// </summary>
+    public class CreateReleaseRequestViewModel
+    {
+        /// <summary>
+        /// TagName
+        /// </summary>
+        [Required]
+        [JsonProperty("tagName")]
+        public string TagName { get; set; }
+
+        /// <summary>
+        /// Name
+        /// </summary>
+        [Required]
+        [JsonProperty("name")]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Body
+        /// </summary>
+        [JsonProperty("body")]
+        public string Body { get; set; }
+
+        /// <summary>
+        /// TargetCommitish
+        /// </summary>
+        [Required]
+        [JsonProperty("targetCommitish")]
```

Comment:
did I see target_comitish used somehwere else in the code? Stardardizing this jsonproperty would be good.

Reply:
RepositoryCreateReleaseOption.cs

Reply:
Good catch regarding target_commitish (that was for the previous specification). Deleting everything that touches that.